vip-plugins/vip-plugins.php

```diff
@@ -0,0 +1,222 @@
+<?php 
+
+/**
+ * Retrieve featured plugins from the vip.wordpress.com API
+ *
+ * @return array an array of plugins
+ */
+function wpcom_vip_fetch_vip_featured_plugins() {
+	$plugins = wp_cache_get( 'wpcom_vip_featured_plugins' );
+
+	if ( false === $plugins ) {
+		$plugins = array();
+		$url_for_featured_plugins = 'https://vip.wordpress.com/wp-json/vip/v1/plugins?type=technology';
+		$response = vip_safe_wp_remote_get( $url_for_featured_plugins, false, 3, 5 );
+
+		if ( ! $response || is_wp_error( $response ) ) {
+			trigger_error( 'The API on vip.wordpress.com is not responding (' . esc_url( $url_for_featured_plugins ) . ')' );
+			return false;
+		}
+
+		$plugins = json_decode( $response['body'] );
+
+		if ( empty( $plugins ) ) {
+			return false;
```

Comment:
Should we trigger a warning here too? Something is probably broken if the response was empty. Also need caching here as well.